app/controllers/banneduser_experiment_controller.py

```diff
@@ -216,8 +216,14 @@ def _update_existing_participants(self, now_utc, modactions):
 
         self.db_session.commit()
 
-    def _get_condition(self):
-        condition = "experienced"
+    def _get_condition(self, newcomer):
+        mapping = {
+            3: "threedays",
+            7: "sevendays",
+            14: "fourteendays",
+            30: "thirtydays"
+        }
+        condition = mapping.get(self._parse_days(newcomer), "unknown")
```

Comment:
this is interesting because we *do* check whether the condition is configured for the experiment later, so it makes sense for there to be only one spot that raises an exception